### ISSUE TITLE
Wrong event fired when clicking confirm button. 

### ISSUE THREAD

**Issue by spacepotato** [t4t5/sweetalert#234]
It may be the case that I am doing something wrong here, but I thought I would list it as an issue in case anyone has had the same problem as me. 

For a bit of context, I am using Sweet Alerts with the Angular wrapper (https://github.com/oitozero/ngSweetAlert) as part of an Ionic App.

The issue I was having was that in a Sweet Alert with both a Confirm and a Cancel button, such as:
```
SweetAlert.swal({
		   title: "Are you sure?",
	           text: "You will no longer be able to view this message",
		   type: "warning",
		   showCancelButton: true,
	           confirmButtonColor: "#DD6B55",
		   confirmButtonText: "Yes, delete it!"
		}, 
		function(isConfirm){ 
		    if(isConfirm){
                       console.log(isConfirm);
		     }
		});
```
Regardless of whether I hit the confirm or the cancel button, isConfirm would be false. I ended up narrowing the issue down to the fact that while hovering over the confirm button set 
targetedConfirm = (target.className === 'confirm')
When I actually clicked on the confirm button the className changed to 'confirm activated'.
I was able to get around this issue by adding:
```
var target = e.target || e.srcElement,
          targetedConfirm    = (target.className === 'confirm activated');
```
After case("click") on line 300 (in the non-minified file), but I feel like this is a bit hacky and was wondering if there is an official solution.

**Comment by JanPeCol** [comment]
It's a annoying bug on mobile platform's.
This workaround works, thanks

**Comment by t4t5** [comment]
This should now be fixed in 0.4.0 by using ```(target.className.indexOf("confirm") !== -1)```
